akka-actor-tests/src/test/scala/akka/io/TcpConnectionSpec.scala

```diff
@@ -364,7 +364,7 @@ class TcpConnectionSpec extends AkkaSpec("""
         ConfigFactory.load(
           ConfigFactory.parseString("akka.io.tcp.direct-buffer-size = 1k")
             .withFallback(AkkaSpec.testConf))
-      override implicit def system: ActorSystem = ActorSystem("respectPullModeTest", config)
+      override lazy val system: ActorSystem = ActorSystem("respectPullModeTest", config)
```

Comment:
It's used by the constructor in base class down the hierarchy of `EstablishedConnectionTest`. A val would give a NPE.

Do you know this Spec? Could you take a look at the failure?